README.md

```diff
@@ -3,15 +3,15 @@
 [travis-img]: https://travis-ci.org/TrySound/rollup-plugin-size-snapshot.svg
 [travis]: https://travis-ci.org/TrySound/rollup-plugin-size-snapshot
 
-This provides the information about
+This plugin provides details about
 
-* actual bundle size which will consume user bundler
-* minified bundle size which will parse browsers in production
-* gzipped size which will be loaded in production
+* actual bundle size
+* minified bundle size
+* gzipped bundle size
```

Comment:
I would like somehow to describe what these three sizes mean for end user and developer. [Here's reference](https://github.com/TrySound/rollup-plugin-size-snapshot/issues/15). Would be great if you will help with this.